tests/cpu/Platform_tests.cpp

```diff
@@ -160,3 +160,17 @@ OCIO_ADD_TEST(Platform, create_temp_filename)
     // Check that it only generates unique random strings.
     OCIO_CHECK_EQUAL(uids.size(), TestMax);
 }
+
+OCIO_ADD_TEST(Platform, utf8_utf16_convert)
+{
+    std::string utf8_str = "\xe3\x81\x93\xe3\x82\x93\xe3\x81\xab\xe3\x81\xa1\xe3\x81\xaf";
```

Comment:
Could you add (as a comment) the letters used to ease the reading?
ex: HIRAGANA LETTER KO UTF-16: x3053, UTF-8: xe3, x81, x93